nixos/tests/mysql-backup.nix

```diff
@@ -23,17 +23,24 @@ import ./make-test.nix ({ pkgs, ... } : {
   testScript =
     '' startAll;
 
+       # delete backup file that may be left over from a previous run
+       $master->execute("rm -f /var/backup/mysql/testdb.gz");
+
        # Need to have mysql started so that it can be populated with data.
        $master->waitForUnit("mysql.service");
 
        # Wait for testdb to be populated.
        $master->sleep(10);
```

Comment:
Is there way to replace this by a command that checks if the database have been added?

Reply:
Good point, let me see..